blueprints/inference/inference-charts/values.yaml

```diff
@@ -52,7 +52,7 @@ inference:
   # Accelerator type: gpu or neuron
   accelerator: gpu
 
-  # Framework type: vllm, rayVllm, or triton-vllm
+  # Framework type: vllm, rayVllm, triton-vllm, or lwsVllm
```

Comment:
Do you think we can name this like vllm, ray-vllm, lws-vllm similar to triton-vllm? We might need a consistency here I think